pepper-apis/src/main/java/org/broadinstitute/ddp/route/PatchFormAnswersRoute.java

```diff
@@ -259,6 +263,9 @@ public Object handle(Request request, Response response) {
             } catch (RequiredParameterMissingException e) {
                 LOG.warn(e.getMessage());
                 throw ResponseUtil.haltError(response, 400, new ApiError(ErrorCodes.REQUIRED_PARAMETER_MISSING, e.getMessage()));
+            } catch (MoreThanOneAnswerForQuestionExistsException e) {
+                LOG.error(e.getMessage());
+                throw ResponseUtil.haltError(response, 500, new ApiError(ErrorCodes.UNEXPECTED_NUMBER_OF_ELEMENTS, e.getMessage()));
```

Comment:
I catch and handle the exception here instead of doing it a the origin because this is the already established pattern in this file. Ideally, it would be nice to refactor the route by catching and handling exceptions closer to places where they were thrown.